Editor/AddressableImportSettings.cs

```diff
@@ -17,12 +17,16 @@ public class AddressableImportSettings : ScriptableObject
     public bool rulesEnabled = true;
 
     [Tooltip("Creates a group if the specified group doesn't exist.")]
-    public bool allowGroupCreation = false;
+    public bool allowGroupCreation;
 
     [Space]
     [Tooltip("Rules for managing imported assets.")]
 #if ODIN_INSPECTOR
-    [ListDrawerSettings(HideAddButton = false,Expanded = false,DraggableItems = true,HideRemoveButton = false)]
+#if ODIN_INSPECTOR
```

Comment:
Seems the `#ODIN_INSPECTOR` should be versioned.

Reply:
Yep, different API in different version.

Reply:
Okay, please change the related symbol to be versioned (`#ODIN_INSPECTOR_X_Y_Z`)

Reply:
@iDreamsOfGame my point is that your patch contains two duplicated `#if ODIN_INSPECTOR` symbol (a typo?), the second one should be a versioned symbol.

```
#if ODIN_INSPECTOR
#if ODIN_INSPECTOR <-- should be versioned like #if ODIN_INSPECTOR_X_Y_Z
...
#else
...
#endif